src/main/java/org/opensearch/neuralsearch/query/NeuralQueryBuilder.java

```diff
@@ -292,15 +292,14 @@ protected QueryBuilder doRewrite(QueryRewriteContext queryRewriteContext) {
             if (vectorSupplier().get() == null) {
                 return this;
             }
-            KNNQueryBuilder knnQueryBuilder = new KNNQueryBuilder(fieldName(), vectorSupplier.get()).filter(filter());
-            if (maxDistance != null) {
-                knnQueryBuilder.maxDistance(maxDistance);
-            } else if (minScore != null) {
-                knnQueryBuilder.minScore(minScore);
-            } else {
-                knnQueryBuilder.k(k);
-            }
-            return knnQueryBuilder;
+            return KNNQueryBuilder.builder()
```

Comment:
This change is to fix neural search package as the main branch is broken and is fixed in this [PR](https://github.com/opensearch-project/neural-search/pull/814/files#diff-7ad8a7a61ba9a613c8cfb89fb01fce11ea496cd687fb3d3edd6eb23383d02e44R304). Therefore ignore this change as while cherrypicking on main it will be taken care off.